Makefile

```diff
@@ -185,7 +185,7 @@ docker-generate-k8s:
 		-v $(shell go env GOCACHE):/root/.cache/go-build:delegated --env GO111MODULE=on \
 		--env https_proxy=$(https_proxy) --env http_proxy=$(http_proxy) \
 		$(BUILD_IMAGE):$(OPERATOR_SDK_VERSION) /bin/bash -c 'operator-sdk generate k8s'
-	cp deploy/crds/* helm/*/crds/
+	cp deploy/crds/* */helm/*/crds/
```

Comment:
generate-k8s generates only zz files, so I think we can get rid of it here

Reply:
yeah I agree, done :)